packages/complex-rubric/controller/src/index.js

```diff
@@ -88,3 +89,119 @@ export const createCorrectResponseSession = (question, env) => {
     }
   });
 };
+
+// this should always be updated when validate function of rubric is changed (as for ebsr)
+const validateSimpleRubric = (model) => {
+  const { points } = model;
+  const errors = {};
+  const pointsDescriptorsErrors = {};
+
+  (points || []).forEach((point, index) => {
+    if(!point|| point === '<div></div>') {
+      pointsDescriptorsErrors[index] = 'Points descriptors cannot be empty.';
+    }
+    else{
+      const identicalPointDescr = points.slice(index + 1).some(p => markupToText(p) === markupToText(point));
+
+      if (identicalPointDescr) {
+        pointsDescriptorsErrors[index] = 'Points descriptors should be unique.';
+      }
+    }
+  });
+
+  if(Object.keys(pointsDescriptorsErrors).length > 0){
+    errors.pointsDescriptorsErrors = pointsDescriptorsErrors;
+  }
+
+  return errors;
+};
+
+// this should always be updated when validate function of multi-trait-rubric is changed (as for ebsr)
```

Comment:
“IMPORTANT! This function is a duplicate of multi-trait-rubric/controller/validate function! If you make any changes to this function, please make sure you also update multi-trait-rubric/controller/validate function!“.


Add this comment (of course, with proper "multi-trait-rubric/controller/validate function") in rubric, multi-trait-rubric and both validation functions in complex-rubric.